### DISCUSSION TITLE
Getting telescope to preview JDTLS content

### DISCUSSION THREAD

**Discussion by LordMZTE** [mfussenegger/nvim-jdtls#586] [Q&A]
I'm having the issue that telescope will not preview `jdt://content/...` URLs, instead just showing an empty window.

Is there anyway I can get telescope to show this?

![screenshot](https://github.com/mfussenegger/nvim-jdtls/assets/28735087/0770f176-7266-4023-9698-48f538d5aa2f)
(The bottom 2 entries are `jdt://content/...` URLs internally, I just wrote a custom `transform_path` function to make them more readable)

- Telescope config: [fennel module](https://git.mzte.de/LordMZTE/dotfiles/src/branch/master/mzte-nv/conf/lua/pluginconf/p-telescope.fnl) and [zig module](https://git.mzte.de/LordMZTE/dotfiles/src/branch/master/mzte-nv/src/modules/telescope.zig)
- nvim-jdtls config: [fennel module](https://git.mzte.de/LordMZTE/dotfiles/src/branch/master/mzte-nv/conf/ftplugin/java.fnl) and [zig module](https://git.mzte.de/LordMZTE/dotfiles/src/branch/master/mzte-nv/src/modules/jdtls.zig)

**Comment by HampusHauffman** [comment]
bump.

**Comment by yopoyka** [comment]
I'm interested in this too.

**Comment by furuochen-dev** [comment]
I'm interested in this too.
